src/style.css

```
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf) format(ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf) format(ttf);
+  font-style: normal;
+  font-weight: bold/800;
```

```suggestion
  font-weight: 800;
```
`bold` or `800`, not both